android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -90,8 +90,21 @@ private Boolean isEmulator() {
   }
 
   private Boolean isTablet() {
-    int layout = getReactApplicationContext().getResources().getConfiguration().screenLayout & Configuration.SCREENLAYOUT_SIZE_MASK;
-    return layout == Configuration.SCREENLAYOUT_SIZE_LARGE || layout == Configuration.SCREENLAYOUT_SIZE_XLARGE;
+    boolean layout = (getReactApplicationContext().getResources().getConfiguration().screenLayout & Configuration.SCREENLAYOUT_SIZE_MASK) >= Configuration.SCREENLAYOUT_SIZE_LARGE;
```

Comment:
Even if `Configuration.SCREENLAYOUT_SIZE_LARGE` is 3 and `SCREENLAYOUT_SIZE_XLARGE;` is 4, I wouldn't rely on that as we never know what may be added in the future..

Can you switch back to testing both values? 🙏

Reply:
Removed the unnecessary else. What do you meen by switching back to test both. Im doing bigger than or equal?
```
 >= Configuration.SCREENLAYOUT_SIZE_LARGE;
```

Reply:
```
int layout = getReactApplicationContext().getResources().getConfiguration().screenLayout & Configuration.SCREENLAYOUT_SIZE_MASK;

if (layout != Configuration.SCREENLAYOUT_SIZE_LARGE && layout != Configuration.SCREENLAYOUT_SIZE_XLARGE) {
  return false;
}

// rest of your code, without the if (layout)
```

Reply:
done